llarp/service/intro.hpp

```diff
@@ -30,7 +30,7 @@ namespace llarp
       }
 
       bool
-      ExpiresSoon(llarp_time_t now, llarp_time_t dlt = 15000) const
+      ExpiresSoon(llarp_time_t now, llarp_time_t dlt = 30000) const
```

Comment:
maybe we should name this instead somewhere. 30000 *what*

Reply:
my eyes are terrible, once a number has more than 2 zeros, I can't tell.  :P 

Is this 3 seconds, 30 seconds or 5 minutes (also `dlt` isn't a great variable name)...